plugins/tar/commands

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+set -eo pipefail; [[ $DOKKU_TRACE ]] && set -x
+source "$(dirname $0)/../common/functions"
+
+case "$1" in
+  tar-from|tar:from)
+    verify_app_name "$2"
+    APP=$2
+    URL=$3
+    shift 3
+    curl -# --insecure -L "$URL" | dokku tar-in $APP "$@"
+    ;;
+
+  tar-in|tar:in)
+    APP="$2"
+    verify_app_name "$2"
+    tee "$DOKKU_ROOT/$APP/src.tar" | wc -c 
```

Comment:
Basically just confirms the total size of the upload tar, it use to be a little progress bar thing, but I couldn't get `awk` to behave.  We could get rid of it and just `>` it into place.